client/snapper.cc

```diff
@@ -371,6 +372,7 @@ command_list(ProxySnappers* snappers, ProxySnapper*)
     const struct option options[] = {
 	{ "type",		required_argument,	0,	't' },
 	{ "all-configs",	no_argument,		0,	'a' },
+	{ "full",	no_argument,	0, 'f' },
```

Comment:
"full" is a bit ambiguous for my taste. Please use ''no-abbrev" like zypper does.